.github/workflows/typespec-validation-all.yaml

```diff
@@ -0,0 +1,74 @@
+name: TypeSpec Validation All
+
+on:
+  push:
+    branches:
+      - main
+      - RPSaaSMaster
+      - typespec-next
+
+  pull_request:
```

Comment:
I removed the PR trigger from the DevOps Pipeline in #31214.  However, I think I like keeping it here now, because it's much faster than TSV when editing core files (for now), so you can use the results of TSVAll in your PR, way before TSV is done.  TSVAll also adds Windows coverage, so even if it's not required, spec authors will look at it.